packages/server/src/api_auth/index.ts

```diff
@@ -463,22 +463,27 @@ router.post('/magic_login', apiRequestAuthMagicLoginValidator,
  * Signup with google
  * POSTMAN_TODO : This route is waiting to be added to postman and documented
  */
-router.post('/google', apiRequestAuthGoogleValidator, async (req, res) => {
+// Helper function for sending error responses
+const sendGErrorResponse = (res: any,status:number,error:string) => {
+    const response:ApiResponse<null> = {
+        success: false,
+        status:status,
+        error:error,
+    };
+    res.status(status).send(response);
+};
+router.post('/google', apiRequestAuthGoogleValidator, async (req,res) => {
+
     // token is required
     try {
         // Validate the request body using the Zod schema
-        const parsedBody = await tokenObjectValidator.parseAsync(req.body);
+        const parsedBody = await tokenObjectValidator.parseAsync(res.locals.reqClientData);
```

Comment:
No need to validate , already validated : `res.locals.reqClientData`. Just use properties directly from `res.locals.reqClientData`